extensions/atlasmap/deployment/src/main/java/org/apache/camel/quarkus/component/atlasmap/deployment/AtlasmapProcessor.java

```diff
@@ -361,6 +75,35 @@ NativeImageResourceBuildItem resource() {
         return new NativeImageResourceBuildItem("META-INF/services/atlas/module/atlas.module");
     }
 
+    @BuildStep
+    void addDependencies(BuildProducer<IndexDependencyBuildItem> indexDependency) {
+        indexDependency.produce(new IndexDependencyBuildItem("io.atlasmap", "atlas-core"));
+        indexDependency.produce(new IndexDependencyBuildItem("io.atlasmap", "atlas-model"));
+        indexDependency.produce(new IndexDependencyBuildItem("io.atlasmap", "atlas-xml-model"));
+        indexDependency.produce(new IndexDependencyBuildItem("io.atlasmap", "atlas-java-model"));
+        indexDependency.produce(new IndexDependencyBuildItem("io.atlasmap", "atlas-json-model"));
+        indexDependency.produce(new IndexDependencyBuildItem("io.atlasmap", "atlas-csv-model"));
+        indexDependency.produce(new IndexDependencyBuildItem("io.atlasmap", "atlas-dfdl-model"));
```

Comment:
Can we open an issue to the atlas map project to include a jandex index ?

Reply:
> Can we open an issue to the atlas map project to include a jandex index ?

+1 for doing it as long as AtlassMap has no index yet. However, that would not eliminate the need for having these IndexDependencyBuildItems, would it?

Reply:
As far as I remember if a dependency includes a jandex index or a beans.xml, then it is automatically indexed but maybe something has changed

Reply:
I'll open one, unless you've already done !!